### PR TITLE
[Maker Cleanup] Remove 'maker-captouch' experiment code (Part 2)

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR removes code related to the 'maker-captouch' experiment that no longer exists. This [prior PR](https://github.com/code-dot-org/code-dot-org/pull/16052) enabled capacitive touch in June 2017 behind an experiment flag because cap touch was causing some performance issues. I found [this forum post](https://forum.code.org/t/capacitive-touch-u6/16216) from May 2018 which states that 'We had to remove the capacitive touch feature because it was causing significant performance issues.' 

This is part 2 of a series of 3 PRs - [Part 1](https://github.com/code-dot-org/code-dot-org/pull/52399) removes code related to the chrome app and [Part 3](https://github.com/code-dot-org/code-dot-org/pull/52435) removes code related to the Maker App pathway.

I confirmed with @dancodedotorg and @moneppo that it is okay to remove code related to the experiment in [this Slack thread](https://codedotorg.slack.com/archives/C0T0EGE8L/p1687531800548269). 

Note that  `TouchSensor.js` is remaining as is code related to touch sensor functions.

## Links

[jira](https://codedotorg.atlassian.net/browse/SL-915)

## Testing story
I tested the CP locally to make sure the board was still working properly.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js [hunk 1]
```diff
@@ -100,13 +99,9 @@ describe('CircuitPlaygroundBoard', () => {
...
 
   describe(`connect()`, () => {
-    // Remove these lines when maker-captouch is on by default.
-    before(() => experiments.setEnabled('maker-captouch', true));
-    after(() => experiments.setEnabled('maker-captouch', false));
-
     it('initializes a set of components', () => {
       return board.connect().then(() => {
-        expect(Object.keys(board.prewiredComponents_)).to.have.length(23);
+        expect(Object.keys(board.prewiredComponents_)).to.have.length(16);
```
Comment (thomasoniii) [inline apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js:107]: What does the magic number mean? Could we bump this to a constant somewhere else?

**Review by thomasoniii — APPROVED** [review]
Deletions are always easy to approve. 🚀 

I had one note about a magic number in a test file - I don't know if there's a better name for it, I don't think a constant like `NUM_PREWIRED_COMPONENTS` is necessarily useful.

Maybe could we put a comment somewhere about why it should be 16 now? That's just the number on the current board, I'm guessing?

**Comment by fisher-alice** [comment]
> Deletions are always easy to approve. 🚀
> 
> I had one note about a magic number in a test file - I don't know if there's a better name for it, I don't think a constant like `NUM_PREWIRED_COMPONENTS` is necessarily useful.
> 
> Maybe could we put a comment somewhere about why it should be 16 now? That's just the number on the current board, I'm guessing?

Thanks Jim! Yes - I'll add a comment that 16 is the number of 'prewired components' on the board. It was 23 if 'captouch' was enabled since there are 7 `TOUCH_PINS` (still defined in `PlaygroundConstants`). This is quite outdated because these pins are from CP Classic. CP Express uses names such as ['A0', 'A1', ...].

`prewiredComponents_` include the initialized components, the board, and the  `JS_CONSTANTS` - assigned [here](https://github.com/code-dot-org/code-dot-org/blob/7d6c9ba5b5e67bf14e234263f503b46046687b04/apps/src/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoard.js#L165).

File: apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js (hunk 1 above)
Comment (fisher-alice) [reply apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js:107]: Added comment.

**Pull request merged**